### ISSUE TITLE
Supporting more `content-type`s for `axum::extract::Json`

### ISSUE THREAD

**Issue by johannescpk** [tokio-rs/axum#391]
### Motivation

Currently `axum::extract::Json` hard-requires the `content-type` of the header to be `application/json`, but an incoming request might hold JSON with a different `content-type` header.

### Proposal

Maybe something like `axum::extract::RelaxedJson` that doesn't check the `content-type` header.

### Workaround

Currently the only workaround I know is manually doing

```rust
let body = hyper::body::to_bytes(body).await.map_err(|_| StatusCode::INTERNAL_SERVER_ERROR)?;
let value: serde_json::Value = serde_json::from_slice(&body).map_err(|_| StatusCode::INTERNAL_SERVER_ERROR)?;
```

**Comment by jplatte** [comment]
This has been done already, but probably not released yet: #378

**Comment by davidpdrsn** [comment]
Yep it'll be shipped in 0.3
